extensions/blocks/podcast-player/edit.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable no-unused-vars */
+
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+import {
+	Button,
+	Disabled,
+	ExternalLink,
+	PanelBody,
+	Placeholder,
+	RangeControl,
+	TextControl,
+	Toolbar,
+	Notice,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { BlockControls, BlockIcon, InspectorControls } from '@wordpress/block-editor';
+import ServerSideRender from '@wordpress/server-side-render';
+import { isURL } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+import { edit, queueMusic } from './icons/';
+import { isAtomicSite, isSimpleSite } from '../../shared/site-type-utils';
+
+const DEFAULT_MIN_ITEMS = 1;
+const DEFAULT_MAX_ITEMS = 10;
+
+const handleSSRError = () => {
+	return <p>{ __( 'Failed to load Block', 'jetpack' ) }</p>;
+};
+
+const PodcastPlayerEdit = ( { attributes, setAttributes } ) => {
+	// Block attributes.
+	const { url, itemsToShow } = attributes;
+
+	// State.
+	const [ editedUrl, setEditedUrl ] = useState( url || '' );
+	const [ editing, setEditing ] = useState( false );
+	const [ urlError, setUrlError ] = useState( '' );
+
+	/**
+	 * Check if the current URL of the Podcast RSS feed
+	 * is valid. If so, set the block attribute and changes
+	 * the edition mode.
+	 * This function is bound to the onSubmit event for the form.
+	 *
+	 * @param {object} event Form on submit event object.
+	 */
+	const checkPodcastLink = event => {
+		event.preventDefault();
+
+		if ( ! editedUrl ) {
+			return;
+		}
+		const isValidURL = isURL( editedUrl );
+
+		setUrlError(
+			! isValidURL
+				? __( 'The URL you entered is invalid. Please check and try again.', 'jetpack' )
+				: ''
+		);
+
+		if ( isValidURL ) {
+			setAttributes( { url: editedUrl } );
+			setEditing( false );
+		}
+	};
+
+	const supportLink =
+		isSimpleSite() || isAtomicSite()
+			? 'http://en.support.wordpress.com/wordpress-editor/blocks/podcast-player-block/'
+			: 'https://jetpack.com/support/jetpack-blocks/podcast-player-block/';
+
+	if ( editing || ! url ) {
+		return (
+			<Placeholder
+				icon={ <BlockIcon icon={ queueMusic } /> }
+				label={ __( 'Podcast Player', 'jetpack' ) }
+				instructions={ __( 'Paste a link to your Podcast RSS feed.', 'jetpack' ) }
```

Comment:
```suggestion
				instructions={ __( 'Enter a URL to your podcast RSS feed.', 'jetpack' ) }
```
(Inspired by the [Google Calendar embed](https://github.com/Automattic/jetpack/blob/76ab061b73ebbf2f42643a68af01d1f9240b66f4/extensions/blocks/google-calendar/edit.js#L104-L105))

Reply:
Reworded to make it clearer. Let's iterate on these in a separate PR.